packages/babel-traverse/src/path/family.js

```diff
@@ -67,6 +67,30 @@ export function getSibling(key) {
   });
 }
 
+export function getPrevSibling() {
+  return this.getSibling(this.key - 1);
+}
+
+export function getNextSibling() {
+  return this.getSibling(this.key + 1);
+}
+
+export function forEachNextSibling(callback = Function.prototype) {
+  let _key = this.key, nextSibling = this.getSibling(++_key);
```

Comment:
To improve readability, could you split theses two lines?

To be consistent in the coding style, `_key` could be then a `const`.